spec/calendar.html

```diff
@@ -526,11 +526,16 @@ <h1>
       <emu-note>For example, week calendar year 2020 includes both 31 December 2019 (a Tuesday belonging to its calendar week 1) and 1 January 2021 (a Friday belonging to its calendar week 53).</emu-note>
     </emu-clause>
 
-    <emu-clause id="sec-buildisomonthcode" aoid="BuildISOMonthCode">
-      <h1>BuildISOMonthCode ( _month_ )</h1>
-      <p>
-        The abstract operation BuildISOMonthCode returns the corresponding string month code, given the numeric ISO calendar month _month_, which must be an integer between 1 and 12, inclusive.
-      </p>
+    <emu-clause id="sec-temporal-isomonthcode" type="abstract operation">
+      <h1>
+        ISOMonthCode (
+          _month_: an integer between 1 and 12 inclusive,
```

Comment:
We should probably use the new phrasing "an integer in the inclusive interval from 1 to 12" going forward, from tc39/ecma262#2848.

Reply:
Good point; do you think I should adopt that here?
```suggestion
          _month_: an integer in the inclusive interval from 1 to 12,
```

Reply:
Yes, we may as well.

Reply:
Done.